htdocs/js/UserList/userlist.js

```diff
@@ -24,4 +24,62 @@
 		export_select_target.addEventListener('change', classlist_add_export_elements);
 		classlist_add_export_elements();
 	}
+
+	// Action form validation.
+	const is_user_selected = () => {
+		const users = document.getElementsByName('selected_users');
+		for (let i = 0; i < users.length; i++) {
+			if (users[i].checked) { return true; }
+		}
```

Comment:
C-style for loops should not be used in modern javascript unless explicitly needed.  Iterating over the elements in an array is not a case where a C-style for loop is explicitly needed.  Change this to:

```suggestion
		for (const user of document.getElementsByName('selected_users')) {
			if (user.checked) return true;
		}
```

Note that the intermediate `users` variable is not needed with this approach.

Also, note the removal of the braces around `return true;`.  Those aren't needed for a single statement inside an `if`.

Please make these changes in the `problemlist.js` and `achievementlist.js` files as well.